pages.pt_BR/linux/snapper.md

```diff
@@ -0,0 +1,36 @@
+# snapper
+
+> Utilitário que cria, compara e reverte snapshots com suporte a linhas de tempo de snapshots automáticos.
+> Mais informações: <https://en.opensuse.org/Portal:Snapper>.
```

Comment:
```suggestion
> Mais informações: <http://snapper.io/manpages/snapper.html>.
```

Reply:
Yes, for sure. Sorry about not reading with attention all the recommendations :)